config.md

```diff
@@ -384,31 +384,84 @@ For POSIX platforms, the configuration structure supports `hooks` for configurin
         * **`env`** (array of strings, OPTIONAL) with the same semantics as [IEEE Std 1003.1-2008's `environ`][ieee-1003.1-2008-xbd-c8.1].
         * **`timeout`** (int, OPTIONAL) is the number of seconds before aborting the hook.
             If set, `timeout` MUST be greater than zero.
+    * **`prestart-runtime`** (array of objects, OPTIONAL) is an array of [prestart-runtime hooks](#prestart-runtime-hooks).
```

Comment:
I'm not sure I follow this. prestart-runtime  hooks look the same as prestart, so why not to keep using prestart hooks?